package.json

```diff
@@ -2,47 +2,56 @@
   "name": "hedwig-react-components",
   "private": true,
   "devDependencies": {
-    "@storybook/addon-actions": "^6.4.22",
-    "@storybook/addon-essentials": "^6.4.22",
-    "@storybook/addon-links": "^6.4.22",
-    "@storybook/addon-viewport": "^6.5.12",
-    "@storybook/react": "^6.4.22",
+    "@storybook/addon-actions": "^6.5.16",
+    "@storybook/addon-essentials": "^6.5.16",
+    "@storybook/addon-links": "^6.5.16",
+    "@storybook/addon-viewport": "^6.5.16",
+    "@storybook/react": "^6.5.16",
```

Comment:
It may be a bit ambitious to upgrade Storybook inside a commit doing something completely different. Storybook does not run for me in this branch.

Reply:
Reverted storybook upgrade